.gitignore

```diff
@@ -1 +1,32 @@
-.vscode/
\ No newline at end of file
+# Prerequisites
+*.d
+
+# Compiled Object files
```

Comment:
why do we need this?

Reply:
.d files are likely dependency-lists, a most handy feature you can utilize between gcc and make to detect changes in no only your source files but also included files

Reply:
Okay. Do we create a .d file anywhere in our build execution?